homepage/static/css/styles.css

```diff
@@ -9,7 +9,7 @@ html, body {
 }
 
 body {
-  font-family: 'Lato', arial, sans-serif;
+  font-family: 'Lato-Light', arial, sans-serif;
```

Comment:
The font [we use](https://github.com/kernc/orange-web/blob/af59a741424a683b5e76815c17bb7a06a3f9448c/homepage/templates/base.html#L18) is [called "Lato"](http://fonts.googleapis.com/css?family=Lato:300,400,700,300italic,400italic,700italic). We load multiple weights of this font, so `font-weight`, if any, is the property to tweak.

With "Lato-Light" nonexistent, this just makes the browser fallback to the next font in sequence.